c2corg_ui/static/partials/mergedocuments.html

```diff
@@ -0,0 +1,31 @@
+<div class="alert alert-warning">
+  <p class="text-center"><strong translate>Warning: This action cannot be undone!</strong></p>
+</div>
+
+<p translate>Merging a source document with a target document transfers all
+associations of the source document to the target document, and sets up a
+redirection from the source to the target document.</p>
+<p><strong translate>Note that comments have to be transferred manually in Discourse before merging.</strong></p>
+
+<div class="document-to-merge">
+  <p>
+    <label>Source:</label> {{ mergeCtrl.sourceDocument.title }}
+    (<a href="/{{ mergeCtrl.module }}/{{ mergeCtrl.sourceDocument.document_id }}"
```

Comment:
`ng-href`([doc](https://docs.angularjs.org/api/ng/directive/ngHref)) instead of href is better maybe?